parser/unparser_formatting_options.go

```diff
@@ -0,0 +1,52 @@
+package parser
+
+import (
+	"fmt"
+
+	"github.com/google/cel-go/common/operators"
+)
+
+// Defined defaults for the formatting options
+var (
+	DefaultWrapColumn = 80
+)
+
+type UnparserFormattingOption func(*formattingOption) (*formattingOption, error)
+
+// Internal implementation of the UnparserFormattingOption interface
+type formattingOption struct {
+	wrapColumn        int // Defaults to 80
+	operatorsToWrapOn map[string]bool
+}
+
+// TODO
+func WrapColumn(col int) UnparserFormattingOption {
+	return func(opt *formattingOption) (*formattingOption, error) {
+		if col < 1 {
+			return nil, fmt.Errorf("Invalid formatting option. Wrap column value must be greater than or equal to 1. Got %v instead", col)
+		}
+		opt.wrapColumn = col
+		return opt, nil
+	}
+}
+
+// Support word wrapping on non-unary symbolic operators. Refer to operators.go for the full list
```

Comment:
nit: should read WrapOnOperators returns an UnparserFormattingOption...

Reply:
Done